src/reducers/existingThoughtMove.js

```diff
@@ -40,13 +40,17 @@ export default (state, { oldPath, newPath, offset }) => {
   const sameContext = equalArrays(oldContext, newContext)
   const oldThought = getThought(state, value)
 
-  /** Find exact thought from thoughtIndex. */
-  const exactThought = () => oldThought.contexts.find(thought => equalArrays(thought.context, oldContext) && thought.rank === oldRank)
+  const isArchived = newThoughts[newThoughts.length - 2] === '=archive'
```

Comment:
Recommend to use the existing `util/isArchived`.

Reply:
`isArchived` takes `Lexeme` as parameter but we only have access to `newPath` in `exisitingThoughtMove`.

Reply:
Of course, my bad.

Regardless, a thought is archived if any of its ancestors is `=archive`, not just the penultimate components of its path.

```js
newThoughts.indexOf('=archive') !== -1
```